packages/ibm-products/src/components/FullPageError/FullPageError.tsx

```diff
@@ -117,7 +110,9 @@ export let FullPageError = React.forwardRef<HTMLDivElement, FullPageErrorProps>(
           <Grid className={`${blockClass}__grid`}>
             <Column sm={4} md={3} lg={6} className={`${blockClass}__column`}>
               <h1 className={`${blockClass}__title`}>
-                <span className={`${blockClass}__label`}>↳ {label}</span>
+                <span className={`${blockClass}__label`} aria-hidden="true">
+                  ↳ {label}
+                </span>
```

Comment:
Ticket #6233 mentions that ↳ symbol is unnecessary for screen readers
But this change will also prevent screen readers from recognising the error label. Could you please revise this to ensure the error label remains accessible to screen readers?